src/main/java/pl/byczazagroda/trackexpensesappbackend/service/WalletServiceImpl.java

```diff
@@ -32,4 +34,12 @@ public WalletDTO createWallet(CreateWalletDTO createWalletDTO) {
         throw new ResourceNotSavedException("Sorry. Something went wrong and your Wallet is not saved. Contact administrator.");
     }
 
+    @Override
+    public WalletDTO findById(Long id) {
```

Comment:
<s>Please add validations:</s>
- <s>id can not be null</s>
- <s>id have to be greater than 0</s>

Variable type can be primitive here so then it can not be `null` anyway.